cityofaustin/atd-vz-data
# Update comprehensive cost widget link

## Associated issues
Closes [#12913](https://github.com/cityofaustin/atd-data-tech/issues/12913) which updates the comprehensive costs by location widget in the VZE dashboard.

## Testing
**URL to test:**  https://deploy-preview-1259--atd-vze-staging.netlify.app/#/dashboard

**Steps to test:**
Check widget link, make sure it goes to this [newest link](https://austin.maps.arcgis.com/apps/instant/interactivelegend/index.html?appid=a43e1b1f94514108be4a91eb0f06a856)

---
#### Ship list
